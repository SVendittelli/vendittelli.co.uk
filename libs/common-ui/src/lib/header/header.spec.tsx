import { render } from '@testing-library/react';

import { BrowserRouter } from 'react-router-dom';
import Header from './header';

describe('Header', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<Header text="Example" />, {
      wrapper: BrowserRouter,
    });
    expect(baseElement).toMatchSnapshot();
  });
});
