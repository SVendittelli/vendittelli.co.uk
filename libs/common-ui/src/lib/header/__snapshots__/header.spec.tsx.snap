// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render successfully 1`] = `
<body>
  <div>
    <header
      class="container"
    >
      <a
        class="title"
        href="/"
      >
        Example
      </a>
      <a
        href="/about-me"
      >
        About Me
      </a>
    </header>
  </div>
</body>
`;
